src/libraries/System.IO.FileSystem/tests/DirectoryInfo/SymbolicLinks.cs

```diff
@@ -52,72 +52,6 @@ protected override void AssertLinkExists(FileSystemInfo link)
             }
         }
 
-        [Theory]
```

Comment:
These tests were validating that we throw when a self-referencing symlink is found in the recursion, while that is true for windows, that wasn't the case for Unix in 5.0, and it was going to be a breaking change caused by the same validation that I'm removing, so I updated the tests to ensure what is expected matches with 5.0